ndcube/ndcube.py

```diff
@@ -667,7 +667,8 @@ def __str__(self):
                 NDCube
                 ------
                 Dimensions: {self.dimensions}
-                Physical Types of Axes: {self.array_axis_physical_types}""")
+                Physical Types of Axes: {self.array_axis_physical_types}
+                Data: Array{self.data.shape} [{self.unit}] '{self.data.dtype}'""")
```

Comment:
```suggestion
                Unit: {self.unit}
                Data Type: {self.data.dtype}""")
```
I suggest his change because the data shape is already captured by the dimensions line, above.